packages/web-runtime/src/components/Topbar/ThemeSwitcher.vue

```diff
@@ -24,9 +24,10 @@ export default defineComponent({
     const currentTheme = computed(() => store.getters.configuration.themes[unref(currentThemeName)])
     const applyTheme = (theme) => {
       for (const param in theme.designTokens.colorPalette) {
-        document
-          .querySelector(':root')
-          .style.setProperty(`--oc-color-${param}`, theme.designTokens.colorPalette[param])
+        ;(document.querySelector(':root') as any).style.setProperty(
```

Comment:
Why do we need to cast this? Shouldn't `document.querySelector` return an `HTMLElement` with `style` attribute?

Reply:
It returns an `Element`, so we need to cast to `HTMLElement`